src/routes.js

```diff
@@ -16,6 +19,26 @@ const routes =  [
                       component: Landing, 
                       name:'Landing' 
                     },
+
+                    {
+                      path: "/auth", 
+                      component: Auth, 
+                       
+                      children: [
+                        {
+                          path: "login", 
```

Comment:
this will be `/auth/login`, we can make it be just `login` without the `auth` by adding `/` before the login. Do the same for the verification `route` too. See screenshot from documentation below
<img width="875" alt="Screenshot 2022-06-29 at 20 37 06" src="https://user-images.githubusercontent.com/8234355/176521893-a6546d18-4eb8-4385-8c54-72cebbf99ca7.png">

```suggestion
                          path: "/login", 
```